runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -296,8 +296,8 @@ private long logUpdateInner(String smrUpdateFunction, final boolean keepUpcallRe
                             + " and we don't have a copy");
                 });
             } catch (TrimmedException ex) {
-                log.warn("getUpcallResultInner: Encountered a trim exception while accessing version {} on attempt {}",
-                        timestamp, x);
+                log.info("accessInner: Encountered trimmed address space " +
```

Comment:
accessInner -> getUpcallResultInner, this seems to be incorrectly copy-pasted.

Reply:
Good catch. Fixed.